Tic_tac_toe.py

```diff
@@ -0,0 +1,159 @@
+def create_board(number_of_rows, number_of_columns):
+    board = []
+    for i in range(0, number_of_rows):
+        board.append(["#"] * number_of_columns)
+    return board
+
+
+def print_board(board):
+    for row in board:
+        print("  ".join(row))
+
+
+def create_players():
+    return ['O', 'X']
+
+
+def get_validated_coordinates(player, board, number_of_rows, number_of_columns):
+    print("Now plays: " + player)
+    input_row = input("Please insert row number: ")
+    input_column = input("Please insert column number: ")
+    if input_row == "" or input_column == "":
+        print("Please insert valid coordinates!")
+        return get_validated_coordinates(player, board, number_of_rows, number_of_columns)
+    move_row = (int(input_row) - 1)
```

Comment:
hmm... shouldn't you add validation, that `input_row` and `input_column` is greater than 0?

Reply:
Done.